src/redux/books/books.js

```diff
@@ -38,20 +24,64 @@ const booksReducer = (state = initialState, action) => {
 };
 
 // ACTIONS CREATOR
-export const addBook = (title, author) => ({
-  type: ADD_BOOK,
-  payload: {
-    id: uuidv4(),
+export const displayBooks = () => async (dispatch) => {
+  const response = await fetch(requestedURL);
+  const res = await response.json();
+  const books = Object.entries(res).map(([key, val]) => {
+    const { title, author, category } = val[0];
+    return {
+      id: key,
+      title,
+      author,
+      category,
+    };
+  });
+  if (books) {
+    dispatch({
+      type: DISPLAY_BOOKS,
```

Comment:
- [ ] Good job so far 👏 As it is among project requirements, we are expected to use this new function provided by the redux toolkit package called [`createAsyncThunk`](https://redux-toolkit.js.org/api/createAsyncThunk) which makes it easier to create Action creators in our React-redux applications. 
![image](https://user-images.githubusercontent.com/90222110/185245981-da3e267e-3b17-40d2-8094-e06fa2e6488b.png)

**Please refactor your action creators in order to use this new way of using the createAsyncThunk** 😉